src/main/app/src/app/taken/taak-view/formio/formio-setup-service.ts

```diff
@@ -235,4 +242,44 @@ export class FormioSetupService {
         ),
     };
   }
+
+  private initializeAvailableDocumentsFieldsetComponent(
+    fieldsetComponent: ExtendedComponentSchema,
+  ): void {
+    if (!this.taak) {
+      return;
+    }
+
+    const documentViewComponent = fieldsetComponent.components?.find(
+      (component: { key: string }) => component.key === "ZAC_Documents",
+    );
+
+    if (!documentViewComponent) {
+      return;
+    }
+
+    fieldsetComponent.type = "fieldset";
+
+    const zoekParameters = new InformatieobjectZoekParameters();
+    zoekParameters.zaakUUID = this.taak.zaakUuid;
+    zoekParameters.gekoppeldeZaakDocumenten = false;
```

Comment:
we should show "all" zaak documents right? what is the `gekoppeldeZaakDocumenten = false` excluding here?

Reply:
@ismailozgen can we remove line 260? I don't think we should care at the moment for linked (or not linked) docs